examples/GeoJSONExporter.js

```diff
@@ -42,6 +47,9 @@ requirejs(['./WorldWindShim',
             wwd.addLayer(layers[l].layer);
         }
 
+        // Create a layer manager for controlling layer visibility.
```

Comment:
Leaving the LayerManager creation at the end works OK. Since it's something prevalent in all our examples, and its functionality is separated from the feature that the examples themselves refer to, we should avoid having it in the middle of the example (which is something that we should address soon in a lot of them).